src/test/ui/rfc-2565-param-attrs/param-attrs-pass.rs

```diff
@@ -0,0 +1,21 @@
+// compile-pass
```

Comment:
Also, this test file doesn't actually check that cfg-stripping occurs semantically. Please use https://rust-lang.github.io/rustc-guide/tests/adding.html#revisions to make sure that this is accounted for.